docs/content/preview/integrations/flyway.md

```diff
@@ -13,23 +13,41 @@ type: docs
 
 [Flyway](https://flywaydb.org/) provides the means to manage schema changes to a YugabyteDB database, among others.
 
-The YugabyteDB-specific implementation of the Flyway APIs has been added to the Flyway community project.
+YugabyteDB has developed a plugin for better integration with Flyway. This plugin is available under the [Flyway Community DB Support](https://github.com/flyway/flyway-community-db-support) project.
 
 ## Prerequisites
 
 To use Flyway with YugabyteDB, you need the following:
 
-- YugabyteDB version 2.4 or later (see [Quick Start](../../quick-start/)).
+- YugabyteDB version 2.20 or later (see [Quick Start](../../quick-start/)).
 
-- Flyway community edition version 7.11.2 or later (see [Download Flyway](https://flywaydb.org/download)).
+- Flyway community edition version 10.12.0 or later (see [Download Flyway](https://flywaydb.org/download)).
+
+- Flyway plugin for YugabyteDB version 10.12.0 or later (see [Maven coordinates](https://central.sonatype.com/artifact/org.flywaydb/flyway-database-yugabytedb)).
 
 - Connection details in the Flyway configuration file `conf/flyway.conf`. Append the following at the end of the file, replacing `flyway.url` with the URL of your running YugabyteDB cluster:
 
   ```properties
-  flyway.url=jdbc:postgresql://localhost:5433/yugabyte
+  flyway.url=jdbc:yugabytedb://localhost:5433/yugabyte
   flyway.user=yugabyte
   flyway.password=yugabyte
   ```
+  - Alternatively, you can also use the vanilla PostgreSQL JDBC Driver instead of the YugabyteDB JDBC Smart Driver. If you use the url with prefix `jdbc:/postgresql://`, the plugin would attempt to use the corresponding driver.
```

Comment:
```suggestion
    You can also use the vanilla PostgreSQL JDBC Driver instead of the YugabyteDB JDBC Smart Driver by setting the URL prefix to `jdbc:/postgresql://`, and the plugin will attempt to use the corresponding driver.
```

Reply:
And then reduce indent of the following code block